### PR TITLE
ADEN-5791 Do not track slot not handled by any provider

### PR DESCRIPTION
**Pull request by fraszczakszymon**

### CODE REVIEW

File: extensions/wikia/AdEngine/js/AdInfoTrackerHelper.js [hunk 1]
```diff
@@ -12,12 +12,11 @@ define('ext.wikia.adEngine.adInfoTrackerHelper',  [
 	var logGroup = 'ext.wikia.adEngine.adInfoTrackerHelper';
 
 	function shouldHandleSlot(slot, enabledSlots) {
-		var dataGptDiv = slot.container.firstChild;
-
 		return (
 			enabledSlots[slot.name] &&
```
Comment (WikiaTech) [inline extensions/wikia/AdEngine/js/AdInfoTrackerHelper.js]: MAJOR Reduce the number of conditional operators (4) used in the expression (maximum allowed 3). [rule](https://sonarqube.wikia-inc.com/coding_rules#rule_key=javascript%3AS1067)

**Comment by WikiaTech** [comment]
SonarQube analysis reported 1 issue
* ![MAJOR][MAJOR] 1 major

Watch the comments in this conversation to review them.

[MAJOR]: https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR'

**Comment by WikiaTech** [comment]
SonarQube analysis reported 2 issues
* ![MAJOR][MAJOR] 2 major

Watch the comments in this conversation to review them.

#### 1 extra issue

Note: The following issues were found on lines that were not modified in the pull request. Because these issues can't be reported as line comments, they are summarized here:

1. ![MAJOR][MAJOR] [AdInfoTracker.js#L9](https://github.com/Wikia/app/blob/2e199bbe4364f1f4381dea324b3931a0e550e097/extensions/wikia/AdEngine/js/AdInfoTracker.js#L9): Remove the unused function parameter "mercuryListener". [rule](https://sonarqube.wikia-inc.com/coding_rules#rule_key=javascript%3AUnusedFunctionArgument)

[MAJOR]: https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR'

**Comment by WikiaTech** [comment]
SonarQube analysis reported 1 issue

Note: The following issues were found on lines that were not modified in the pull request. Because these issues can't be reported as line comments, they are summarized here:

1. ![MAJOR][MAJOR] [AdInfoTracker.js#L9](https://github.com/Wikia/app/blob/929ff5dcb0c3217d79353d708707d05bc1979e41/extensions/wikia/AdEngine/js/AdInfoTracker.js#L9): Remove the unused function parameter "mercuryListener". [rule](https://sonarqube.wikia-inc.com/coding_rules#rule_key=javascript%3AUnusedFunctionArgument)

[MAJOR]: https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR'

**Comment by WikiaTech** [comment]
SonarQube analysis reported 1 issue

Note: The following issues were found on lines that were not modified in the pull request. Because these issues can't be reported as line comments, they are summarized here:

1. ![MAJOR][MAJOR] [AdInfoTracker.js#L9](https://github.com/Wikia/app/blob/22aacb830ec8310ed404fb5c77ea65e7e5dbec35/extensions/wikia/AdEngine/js/AdInfoTracker.js#L9): Remove the unused function parameter "mercuryListener". [rule](https://sonarqube.wikia-inc.com/coding_rules#rule_key=javascript%3AUnusedFunctionArgument)

[MAJOR]: https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR'

**Review by kvas-damian — APPROVED** [review]

**Pull request merged**